app/views/summaries/index.html.erb

```diff
@@ -1,8 +1,10 @@
+<h1 class="title header-centered ">Total Spending</h1>
+<%= pie_chart @totals, donut: true, legend: false  %>
 <% if Purchase.any? %>
```

Comment:
I don't think you need this table at all.

Reply:
Edit: keep the "Total spending to date" line, but assign it in the controller e.g. `@overall_sum`